ui/src/components/credentials/IssueCredentialForm.tsx

```diff
@@ -213,182 +215,266 @@ export function IssueCredentialForm({
     [env, fetchJsonSchema, initialValues.schemaID, messageAPI]
   );
 
+  const computeFormObjectInitialValues = (
+    objectAttribute: ObjectAttribute,
+    initialValues: Record<string, unknown>
+  ): Record<string, unknown> | undefined => {
+    return objectAttribute.schema.attributes?.reduce(
+      (acc: Record<string, unknown>, curr: Attribute): Record<string, unknown> => {
+        switch (curr.type) {
+          case "boolean": {
+            const parsedConst = z.boolean().safeParse(curr.schema.const);
```

Comment:
Why are we parsing the `const` and `default` values as `unknown` in the `JsonSchema` parser and then we are parsing them again here to `boolean`, `number` or `string`? I think that we should already parse them correctly in the `JsonSchema` parser.

Reply:
The [JSON Schema standard allows the constant and default values to have any shape](http://json-schema.org/understanding-json-schema/reference/generic.html#annotations), including shapes that do not validate against the schema. This means that valid schemas can include non-valid values in these keywords. Applying this proposal would make valid schemas fail zod validation.

Reply:
Mmmm maybe I explained myself wrong. What I wanted to say is to move this code to a zod `transform` when parsing the `JsonSchema`. It shouldn't fail, but we can add a `defaultValue` to the `Attribute` type and just set it there.

Reply:
Let's discuss this at a meeting because I don't see how this can work.

Reply:
We agreed to create another issue to consider these changes.